src/frontend/js/components/PaginateCourseSearch/index.tsx

```diff
@@ -123,7 +123,8 @@ export const PaginateCourseSearch = ({ courseSearchTotalCount }: PaginateCourseS
                 </li>
               ) : (
                 <li className="pagination__item">
-                  <a
+                  <button
+                    tabIndex={index}
```

Comment:
We're not supposed to use tabIndex with positive values at all. Here the effect is those buttons are the first items in the tab order before anything else on the page, which I don't think is what we want.

I think this article sums up what I know about tab indexes pretty well: https://bitsofco.de/how-and-when-to-use-the-tabindex-attribute/